static/js/publisher/form/fileInput.js

```diff
@@ -186,7 +187,7 @@ class FileInput extends React.Component {
         className={localClassName.join(" ")}
         onClick={this.fileClickHandler}
         onKeyDown={this.keyboardEventHandler}
-        tabIndex={0}
+        tabIndex={noFocus ? null : 0}
```

Comment:
I find bool props that are negations quite confusing and harder to comprehend.
Could it be 'isFocusable' that is `true` by default and can be turned off if needed?